src/openrct2/object/Object.h

```diff
@@ -167,12 +168,13 @@ struct rct_object_entry_group
 assert_struct_size(rct_object_entry_group, 8);
 #endif
 
+#pragma pack(pop)
```

Comment:
Does this still make sense?

Reply:
Yes. `rct_object_entry` is still a fixed sized thing that is read straight into memory I think.

Reply:
Some of the definitions in this file might still need it. Though there are also some unused definitions here. I will do a cleanup in a future PR if nobody beats me to it.